app/res/values/strings.xml

```diff
@@ -107,6 +107,7 @@ Copyright (c) 2012 readyState Software Ltd.</string>
     <string name="altitude">Altitude</string>
     <string name="app_name">ODK Collect</string>
     <string name="attachment_oversized">Warning: attached large file of size %s kb. This could cause problems with submitting to the server depending on your network conditions.</string>
+    <string name="attachment_invalid">Invalid filetype choosen. Please select a valid multimedia file.</string>

```

Comment:
add this string to messages_ccodk_default.txt, call Localization.get() on the message key